ENIGMAsystem/SHELL/Platforms/SDL/Window.cpp

```diff
@@ -113,7 +115,11 @@ int handleEvents() { return eventHandler.processEvents(); }
 
 namespace enigma_user {
 
-void io_handle() {}
+void io_handle() {
```

Comment:
this can prolly be used as io_handle for all systems but I'll do that in a separate commit